mappings/net/minecraft/client/world/ClientWorld.mapping

```diff
@@ -6,6 +6,9 @@ CLASS net/minecraft/class_638 net/minecraft/client/world/ClientWorld
 	FIELD field_18226 players Ljava/util/List;
 	FIELD field_21526 lightningTicksLeft I
 	FIELD field_21527 colorCache Lit/unimi/dsi/fastutil/objects/Object2ObjectArrayMap;
+	FIELD field_24430 levelProperties Lnet/minecraft/class_638$class_5271;
```

Comment:
`clientWorldProperties` maybe, as parent class already exposes a `WorldProperties` protected final field

Reply:
Yeah definitely in that case